packages/l10n/utils.js

```diff
@@ -0,0 +1,41 @@
+export const mapLocaleToIntlLocale = locale => {
+  if (locale.startsWith('de')) return 'de';
+  if (locale.startsWith('es')) return 'es';
+  if (locale.startsWith('fr')) return 'fr-FR';
+  if (locale === 'zh-CN') return 'zh-CN';
+  return 'en';
+};
+
+const supportedLocales = ['en', 'de', 'es', 'fr-FR', 'zh-CN'];
+const defaultLocale = 'en';
+
+// Given a locale, return the locale only if it's whitelisted.
+// If not, return a default locale.
+export function getSupportedLocale(locale) {
+  const isSupported = supportedLocales.find(supportedLocale =>
```

Comment:
Ah okay. Just to recap for me: given we have a `language` not really a `locale` in `supportedLocales` (like `es`) we whitelist all of their corresponding `locales` (as they start with `es`).

Reply:
Yes, I thought this was the best approach to handle both formats

Reply:
Sure. I was just thinking out loud.